### PR TITLE
Fix bicycle example

### PR DESCRIPTION
**Pull request by moorepants**
#### Brief description of what is fixed or changed

Updated the documentation example for the linear bicycle model in mechanics so that it works with modern SymPy code and doctests the resulting eigenvalues.

#### Release Notes

* physics.mechanics
  * Updated the bicycle model example in the documentation to produce correct results and improve the explanation and formatting.

### CODE REVIEW

**Comment by moorepants** [comment]
Passes doctests in the CI: `sympy/doc/src/tutorials/physics/mechanics/bicycle_example.rst [94]          [OK]`

**Comment by moorepants** [comment]
@eh111eh would you like to review this?

File: doc/src/tutorials/physics/mechanics/bicycle_example.rst [hunk 1]
```diff
@@ -1,94 +1,104 @@
...
   >>> TempFrame = Frame.orientnew('TempFrame', 'Axis', [-htangle, Frame.y])
   >>> Fork = Frame.orientnew('Fork', 'Axis', [q5, Frame.x])
   >>> TempFork = Fork.orientnew('TempFork', 'Axis', [-htangle, Fork.y])
-  >>> WF = ReferenceFrame('WF')
+  >>> WF = me.ReferenceFrame('WF')
 
+This block of code is forming the positions of the relevant points rear wheel
+contact -> rear wheel's center of mass -> frame's center of mass + frame/fork
+connection -> fork's center of mass + front wheel's center of mass -> front
+wheel contact point. ::
```
Comment (smichr) [inline doc/src/tutorials/physics/mechanics/bicycle_example.rst:78]:
```suggestion
Define relevant points for the system
-------------------------------------

``WR_cont`` - rear wheel contact
``WR_mc``- rear wheel's center of mass
``Steer`` - frame/fork connection ??? this is unclear
``Frame_mc`` - frame's center of mass
``Fork_mc`` - fork's center of mass
``WF_mc`` - front wheel's center of mass
``WF_cont`` - front wheel contact point
```

Comment (smichr) [reply doc/src/tutorials/physics/mechanics/bicycle_example.rst:78]: I'm not sure what point Steer represents.

File: doc/src/tutorials/physics/mechanics/bicycle_example.rst [hunk 2]
```diff
@@ -1,94 +1,110 @@
...
   >>> TempFork = Fork.orientnew('TempFork', 'Axis', [-htangle, Fork.y])
-  >>> WF = ReferenceFrame('WF')
+  >>> WF = me.ReferenceFrame('WF')
+
+Define relevant points for the system
+-------------------------------------
+
+``WR_cont`` - rear wheel contact
+``WR_mc``- rear wheel's center of mass
+``Steer`` - frame/fork connection ??? this is unclear
```
Comment (moorepants) [inline doc/src/tutorials/physics/mechanics/bicycle_example.rst:80]:
```suggestion
``Steer`` - frame/fork connection
```

**Pull request merged**